Q/QuantumEspresso/build_tarballs.jl

```diff
@@ -80,4 +80,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               preferred_gcc_version=v"5", julia_compat="1.6", preferred_llvm_version=v"11")
+               preferred_gcc_version=v"5", julia_compat="1.6")
```

Comment:
Let's see whether gcc 6 makes any difference:
```suggestion
               preferred_gcc_version=v"6", julia_compat="1.6")
```

Reply:
yeah, good idea.